### ISSUE TITLE
Vendor Showcase

### ISSUE THREAD

**Issue by mlwoodward** [woodwarddesign/GWG-Clothing#9]
The client would like a product and company profile for vendors.  They referenced kryptek.com.  See screenshot.  Will get the client an estimate for the project.

![gwg-vendor](https://cloud.githubusercontent.com/assets/4562222/4566922/fdd74df8-4f2c-11e4-8468-c746d0b9e89b.jpg)

**Comment by gwgjan** [comment]
This project has escalated with our manufacturer requesting this ASAP. Please provide an estimate. I'd like to go over what we can do on our retail portal already to see if there is a way to tie it in or if this needs to be it's own page.

**Comment by mlwoodward** [comment]
**From Jen..

Please see my attachment showing you a rough idea of what we want to have done.  Here are some of the items that we will need to have download area for.  Please let me know if you have any questions, or if you would like to look at the Kryptek Portal again.  I can provide you with their information.

Front page Twitter, Instagram, Facebook, Pintrest, & You Tube
Front page link to our blog (there is one currently there that could be done better)
Download Headers as follows:
In Store Graphics
Promotional Photos
Catalogs
Brand Guidelines
Spec Sheets
How to Become a Retailer
GWG Lifestyle
Our Partners
Company (This is where we want to include our Mission Statement, Our About Us with a photos and Warranty and return info.
I think our classic black and pink is a good look, but the photos will mostly be done with a clean white background.  Norissa and Jan are both included in this.  Please add anything I may have missed.  We are heading out of the country again on Sunday, so please let me know if we need to talk beforehand.  We will not be able to approve the final on this until we return the 11th, so if it can be done by then that would be great, but no rush.  However, I would like to know when the site assessment will be done?  I am anxiously awaiting moving forward on some projects as we are fast approaching our busy season.  I look forward to your reply.  Thank you.

Jen

**Comment by thomas-thackery** [comment]
![gwg-retailer-layout](https://cloud.githubusercontent.com/assets/536441/4812711/d973ae52-5ec2-11e4-9c88-23f77266a0e1.png)

**Comment by thomas-thackery** [comment]
:question: **RE: Front page download region with blocks**
What types of the downloadable assets will included? pdfs, zip file (collection), single image combo of all?

**Comment by gwgjan** [comment]
Yes to a combination of all of these. We will also need to load excel files. I'm not sure about Word Docs but it might be good to include that option. I'm assuming this will also include .jpgs and .pngs

**Comment by gwgjan** [comment]
Please let me know if any of these pose a difficulty and we can review.

**Comment by thomas-thackery** [comment]
@mike 
- I resized and set bg. It displays best as fixed rather than contained so that the the GWG displays to the left. Still looks good responsive
- Main nav on original site is image based. I made this CSS w/ 'to bottom' gradient which will contribute to faster page load. 
- Need to position logo to top left, might need to adjust template file (lets talk on that).

**Comment by thomas-thackery** [comment]
@gwgjan 
Shoot me (or Mike) a couple of sample images/files/pdf and I'll use them for testing. :gun:

**Comment by gwgjan** [comment]
@kveldalf I just sent files to Mike's email address since I don't have yours.

**Comment by mlwoodward** [comment]
@gwgjan - I just received the one photo.  See attached image below.  Where there others?
![dsc_2815](https://cloud.githubusercontent.com/assets/4562222/4851104/cfb2f55e-606f-11e4-9ddc-b6a8130b60e2.jpg)

**Comment by gwgjan** [comment]
There was a link to 6 images from dropbox. It may not have worked. I will send more individually.

**Comment by mlwoodward** [comment]
I will check that shared folder.

**Comment by mlwoodward** [comment]
@kveldalf here are all the photos.  Let me know if you want them sent via wetransfer or something.
![dsc_2815](https://cloud.githubusercontent.com/assets/4562222/4851682/cb628942-6074-11e4-8207-c48c870c2fe1.jpg)
![dsc_2830](https://cloud.githubusercontent.com/assets/4562222/4851683/cb640984-6074-11e4-8cb8-9e67102dc63d.jpg)
![dsc_2849](https://cloud.githubusercontent.com/assets/4562222/4851686/cb68c3fc-6074-11e4-8622-558aa45c5e45.jpg)
![dsc_2867](https://cloud.githubusercontent.com/assets/4562222/4851685/cb681ea2-6074-11e4-906c-5d1785db44b0.jpg)
![dsc_5597](https://cloud.githubusercontent.com/assets/4562222/4851687/cb6d04a8-6074-11e4-9cc8-3ad4076f9a8c.jpg)
![dsc_5633](https://cloud.githubusercontent.com/assets/4562222/4851684/cb65ab04-6074-11e4-8dde-fe8f89715028.jpg)

**Comment by thomas-thackery** [comment]
@gwgjan Do you have a high resolution logo image? One at least at 220dpi. The current logo shows signs of pixelation on retina display
![](https://www.gwgclothing.com/skin/frontend/3295/retailer/images/logo.png)

**Comment by gwgjan** [comment]
@kveldalf Will this work? I have it in a .pdf format but I needed to remove the ™ since we are now ®. Will we be able to easily change out the logo? We have an updated logo in the works
![gwg_logo-color](https://cloud.githubusercontent.com/assets/8901784/4866985/19438032-6131-11e4-80cb-5166fba08c0a.png)

**Comment by thomas-thackery** [comment]
@gwgjan :+1: This will work. Thanks! Swapping out with new logo should be straight forward.
